dlt/destinations/dataset/ibis_relation.py

```diff
@@ -186,6 +253,76 @@ def select(self, *columns: str) -> "ReadableIbisRelation":
         """set which columns will be selected"""
         return self._proxy_expression_method("select", *columns)  # type: ignore
 
+    # TODO ensure same defaults in ReadableDBAPIDataset and ReadableIbisRelation; and docstrings
+    def list_load_ids(
+        self, status: Union[int, list[int], None] = 0, limit: Optional[int] = None
+    ) -> "ReadableIbisRelation":
+        load_table = self._dataset.table(self.schema.loads_table_name)
+        if status is not None:
+            status = [status] if isinstance(status, int) else status
+            load_table = load_table.filter(load_table["status"].isin(status))
```

Comment:
it would be interesting to find out wether `isin` with a one item list is the same speed as filter with that value. I'd hope the sql optimizer would find this automatically. Just a thought.